### PR TITLE
Set up the rails api app and add linter files

### PR DESCRIPTION
**Pull request by Sonickmumba**
In this pull request, I implement the Rails API application as the requirement given below:

- Create a new Rails API app called 'hello-rails-back-end'.
- Initialize your project with Git.
- Make sure that your project has a Postgres database set up.
- Create a table for storing your messages and create 5 different greetings.
- Create an API endpoint that selects a random greeting from your table (you will need a controller with an action an rails route).

### CODE REVIEW

**Review by rotimiazeez — COMMENTED** [review]
# Hi @Sonickmumba , 

### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Please ensure In the README file for the backend, you should find a link to the frontend part and In the README file for the frontend, you should find a link to the backend part.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Comment by Sonickmumba** [comment]
@rotimiazeez thanks for much. Do you mean the links to the individual project repo ?

**Comment by rotimiazeez** [comment]
@Sonickmumba yes please

File: config/initializers/cors.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Be sure to restart your server when you modify this file.
+
+# Avoid CORS issues when API is called from the frontend app.
+# Handle Cross-Origin Resource Sharing (CORS) in order to accept cross-origin AJAX requests.
+
+# Read more: https://github.com/cyu/rack-cors
+
```
Comment (danielufeli) [inline config/initializers/cors.rb:7]: - [ ] Nicely done, kindly check I had Issues with `cors` while testing your project, I had to un-comment the code in this `cors.rb` file and `gem "rack-cors"` in your gem file to make it work.

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @Sonickmumba , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Nice commits messages✔️
- Clean Code ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/initializers/cors.rb (hunk 1 above)
Comment (Sonickmumba) [reply config/initializers/cors.rb:7]: @danielufeli thank you so much. What exactly am I suppose to do? I am lost. Do I have to also uncomment the code in core.rb and gem "rack-cors" in the gem file?

**Review by danielufeli — APPROVED** [review]
Hi @Sonickmumba ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

ChopApproveCopApproveGIF

## Highlights
- Nice commits messages✔️
- Clean Code ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._ate_comments_counter`

**Pull request merged**